src/sentry/static/sentry/app/components/selectMembers/index.tsx

```diff
@@ -57,6 +57,7 @@ type Props = {
   onInputChange?: (value: any) => any;
   disabled?: boolean;
   placeholder?: string;
+  styles?: Object;
```

Comment:
"""suggestion
    styles?:  React.CSSProperties;
"""

Reply:
The properties we're supplying turns out to not be part of React.CSSProperties. Should we revert to using object, narrowing the type to ` { control: (provided: any) => any; }` or widening CSSTypes ?